### PR TITLE
 Leather gloves and an LED tube light.

### PR DESCRIPTION
**Pull request by PrPleGoo**
Added BurnTemperature to bulbs.
Added HeatResistance to clothing and species.
Added HeatResistanceComponent which resolves armor vs skin.
Made the hand burn on lamps only happen when heat resistance is too poor.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -279,3 +279,4 @@ BuildFiles/Windows/Godot/*
 
 # Working on the tools scripts is getting annoying okay?
 .mypy_cache/
+.gitmodules
```
Comment (PJB3005) [inline .gitignore]: I think you accidentally re-committed these changes to the submodule.

File: .gitmodules [hunk 2]
```diff
@@ -1,4 +1,4 @@
 [submodule "RobustToolbox"]
 	path = RobustToolbox
-	url = https://github.com/space-wizards/RobustToolbox.git
+	url = https://github.com/PrPleGoo/RobustToolbox.git
```
Comment (PJB3005) [inline .gitmodules]: Just so you know, you don't need to modify this file to check out your fork. Once the submodule is initialized you can run the normal `git remote` commands in it and it won't affect the master repo.

File: Content.Server/GameObjects/Components/Power/PoweredLightComponent.cs [hunk 3]
```diff
@@ -141,9 +154,9 @@ public void UpdateLight()
             switch (LightBulb.State)
             {
                 case LightBulbState.Normal:
-                    device.Load = Load;
                     if (device.Powered)
                     {
+                        device.Load = Load;
```
Comment (PJB3005) [inline Content.Server/GameObjects/Components/Power/PoweredLightComponent.cs]: I'm pretty sure this change is gonna re-introduce the bug where lights go strobe light mode when running out of power. Why are you changing it?

Comment (PrPleGoo) [reply Content.Server/GameObjects/Components/Power/PoweredLightComponent.cs]: Tested it, didn't know that was a thing. Reverted the change.

File: .gitignore (hunk 1 above)
Comment (PJB3005) [reply .gitignore]: Still this gitignore change in here

**Comment by PrPleGoo** [comment]
> Looks good. However, in the future someone might want to change LightBulbs so LED bulbs consume less electricity than regular bulbs.

**Pull request merged**